src/components/Layout/Header/TxWindow/TxWindow.tsx

```diff
@@ -0,0 +1,149 @@
+import type { TabProps } from '@chakra-ui/react'
+import {
+  Box,
+  Circle,
+  Drawer,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  IconButton,
+  Select,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@chakra-ui/react'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import React, { useMemo, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { TxHistoryIcon } from 'components/Icons/TxHistory'
+import { RawText } from 'components/Text'
+import { TransactionsGroupByDate } from 'components/TransactionHistory/TransactionsGroupByDate'
+import { selectTxIdsByFilter } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+const padding = { base: 4, md: 6 }
+
+const CustomTab: React.FC<TabProps> = props => (
+  <Tab
+    px={0}
+    fontWeight='medium'
+    color='text.subtle'
+    cursor='pointer'
+    _selected={{ color: 'text.base' }}
+    _hover={{ cursor: 'pointer' }}
+    {...props}
+  />
+)
+type TxsByStatusProps = {
+  txStatus: TxStatus
+  limit: string
+}
+const TxsByStatus: React.FC<TxsByStatusProps> = ({ txStatus, limit }) => {
+  const translate = useTranslate()
+  const filter = useMemo(() => ({ txStatus }), [txStatus])
+  const txIds = useAppSelector(state => selectTxIdsByFilter(state, filter)).slice(0, Number(limit))
```

Comment:
`.slice()` would yield a new reference every render, let's have the sliced version in a separate memoized const